xrpl-hooks-docs/md/hooks-guard-call-non-const.md

```diff
@@ -0,0 +1,4 @@
+# hooks-guard-call-non-const
+
+Only compile-time constants can be used as an argument in loop GUARD call. This checker prints warning if a non compile-time constant is used. 
```

Comment:
Well, it doesn't really print - the warning is passed from clangd all the way to the Hooks Builder window in more structured formats... Existing docs generally say just "warns"... Also, it's "check" rather than "checker". Also, there could be a link to https://xrpl-hooks.readme.io/docs/loops-and-guarding , which describes the reason for the check in more detail...